### PR TITLE
Export namespace with TypeScript types

### PR DESCRIPTION
**Pull request by bensalilijames**
As discussed in: https://github.com/helmetjs/helmet/issues/209

### CODE REVIEW

File: index.ts [hunk 1]
```diff
@@ -1,22 +1,28 @@
 import { IncomingMessage, ServerResponse } from 'http';
 
-interface DnsPrefetchControlOptions {
-  allow?: boolean;
+// eslint-disable-next-line @typescript-eslint/no-namespace
```
Comment (bensalilijames) [inline index.ts:3]: Would it be worth disabling this in `eslint-config-helmet`?

**Comment by EvanHahn** [comment]
I think we already discussed this, but just to double-check:

If we have `export =` in this file, we can't do something like `export interface DnsPrefetchControlOptions`, right? We can only have one export?

I'll give this a more thorough review soon.

**Comment by bensalilijames** [comment]
Yep, that's exactly right! 👍

**Comment by EvanHahn** [comment]
I plan to solve this by moving everything into a single repo, in helmetjs/helmet. `helmet@4` should export these types properly.

Closing this PR, but it will "live on" in the Helmet repo.

**Pull request closed without merging**